### PR TITLE
Fix a few lgtm warnings

### PR DESCRIPTION
**Pull request by Hi-Angel**
source: https://lgtm.com/projects/g/qutebrowser/qutebrowser/alerts/?mode=tree

There're more warnings, in this PR mostly trivial stuff.

---
This change is [Reviewable](https://reviewable.io/reviews/qutebrowser/qutebrowser/4676)

### CODE REVIEW

File: misc/userscripts/qute-keepass [hunk 1]
```diff
@@ -148,7 +148,7 @@ def dmenu(items, invocation, encoding):
 
 def get_password():
     """Get a keepass db password from user."""
-    _app = QApplication(sys.argv)
+    QApplication(sys.argv)
```
Comment (jgkamat) [inline misc/userscripts/qute-keepass]: I think we need to keep a handle to this or nothing will work, this change makes this script crash for me.

File: qutebrowser/mainwindow/mainwindow.py [hunk 2]
```diff
@@ -584,8 +584,7 @@ def closeEvent(self, e):
         quit_texts = []
         # Ask if multiple-tabs are open
         if 'multiple-tabs' in config.val.confirm_quit and tab_count > 1:
-            quit_texts.append("{} {} open.".format(
-                tab_count, "tab is" if tab_count == 1 else "tabs are"))
```
Comment (jgkamat) [inline qutebrowser/mainwindow/mainwindow.py]: Doesn't this change the behavior?

File: misc/userscripts/qute-keepass (hunk 1 above)
Comment (Hi-Angel) [reply misc/userscripts/qute-keepass]: Thx, I re-wrote this commit to put a comment making sure nobody would accidentally remove this variable.

**Comment by Hi-Angel** [comment]
Thanks! I don't have write access, could please someone merge it for me?

File: misc/userscripts/qute-keepass [hunk 3]
```diff
@@ -148,7 +148,7 @@ def dmenu(items, invocation, encoding):
 
 def get_password():
     """Get a keepass db password from user."""
-    _app = QApplication(sys.argv)
+    _app = QApplication(sys.argv) # don't remove this local variable
```
Comment (toofar) [inline misc/userscripts/qute-keepass]: Not sure if we lint these userscripts but there is some PEP that says there needs to be two spaces before comments on the same line as code.

Comment (Hi-Angel) [reply misc/userscripts/qute-keepass]: Np, done.

**Comment by The-Compiler** [comment]
LGTM! @Hi-Angel, do you plan to look at the other issues pointed out there as well? Some of them (mostly "unused" imports and `raise utils.Unreachable`) are bogus, but there are some more which seem genuine and you haven't fixed here :wink:

**Comment by Hi-Angel** [comment]
Thanks @The-Compiler! No, I currently don't. After I found the site I'm doing a sweep over projects I'm interested in, and since I'm not so familiar with Python as I am with C/C++, I'd rather prioritize C/C++-based projects. I already sent some patches that I'm gonna need to reply to this evening; and then there's more warnings in those projects… So, it's unlikely that I get back to other python warnings any time soon, the current PR contains trivial stuff that doesn't even require Python knowledge.

**Comment by The-Compiler** [comment]
Thanks! I opened #4678 to look at the rest.

**Pull request merged**